.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: objective-c
+osx_image: xcode7.3
+
+script:
+  - chmod u+x bootstrap.sh
```

Comment:
What do you think about letting Git track this instead? I.e. commit the file with the executable bit and then not having to set it after cloning.